frontend/components/common/Grid.js

```diff
@@ -1,6 +1,69 @@
+import React from 'react'
```

Comment:
@ezwang why do you think this is showing up as changed? This code was already there before this PR I thought...

Reply:
no, I think it was added here. Also there has been a lot of commit duplication due to people not updating their branches properly after another person force pushes, so this might be another result of that otherwise

Reply:
@cacabo A lot of the changed code was already here! It seems like one of the merges went wrong early on. I cleaned up the backend duplication issues, but the frontend duplication issues still remain.